compute_tools/Cargo.toml

```diff
@@ -18,9 +18,11 @@ clap.workspace = true
 flate2.workspace = true
 futures.workspace = true
 hyper0 = { workspace = true, features = ["full"] }
+metrics.workspace = true
 nix.workspace = true
 notify.workspace = true
 num_cpus.workspace = true
+once_cell.workspace = true
```

Comment:
Doesn't the Rust standard library have a once cell type thing now?

Reply:
https://doc.rust-lang.org/std/cell/struct.LazyCell.html

Reply:
LazyCell is not thread safe, so I would need LazyLock

if you don't mind, I will keep `once_cell`, because that's what we already use in the repository.